jangjuseong/models/likeDao.js

```diff
@@ -0,0 +1,21 @@
+const mysqlDataSource = require('./daoModule');
+
+const postLike = async (userId, postId) => {
```

Comment:
likeDao의 함수명도 수정하면 좋을 것 같습니다.

```suggestion
const createLike = async (userId, postId) => {
```